src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -196,6 +196,7 @@ const DomainFragment = ({
     const renderHeader = (data, index) => {
       return (
         <DiscoveryTitleSeparator
+          withBorderBottom={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change seems to be fine as it adds a new prop `withBorderBottom` to the `DiscoveryTitleSeparator` component. However, it would be better if you could add a default value for this prop in the `DiscoveryTitleSeparator` component itself. This way, even if the prop is not provided, the component will have a fallback and won't break.
```diff
+ withBorderBottom={true}
```
This can be done like so in the `DiscoveryTitleSeparator` component:
```javascript
DiscoveryTitleSeparator.defaultProps = {
  withBorderBottom: false,
};
```
This sets the default value of `withBorderBottom` to `false` if it's not provided when the component is used.


<!-- This is an auto-generated comment by OSS CodeRabbit -->